Cannot install webpack-cli using npm

I'm trying to install webpack-cli with npm, but I'm getting error code 128.

Thanks in advance

Sure, here it is:

> sudo npm install -g webpack-cli
> Password:
> npm ERR! Error while executing:
> npm ERR! /usr/bin/git ls-remote -h -t git://github.com/kalcifer/recast.git
> npm ERR!
> npm ERR! fatal: unable to connect to github.com:
> npm ERR! github.com[0: 192.30.253.113]: errno=Operation timed out
> npm ERR! github.com[1: 192.30.253.112]: errno=Operation timed out
> npm ERR!
> npm ERR!
> npm ERR! exited with error code: 128

> npm ERR! A complete log of this run can be found in:
> npm ERR!     /Users/.../.npm/_logs/2017-09-01T14_53_03_375Z-debug.log

I get similar error:

```cmd
> npm install webpack-cli@latest
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t git://github.com/ev1stensberg/generator.git
npm ERR!
npm ERR! fatal: unable to connect to github.com:
npm ERR! github.com[0: 192.30.253.112]: errno=No such file or directory
npm ERR! github.com[1: 192.30.253.113]: errno=No such file or directory
npm ERR!
npm ERR!
npm ERR! exited with error code: 128

npm ERR! A complete log of this run can be found in:
```

--- a/package.json
+++ b/package.json
@@ -93,7 +93,7 @@
     "webpack-addons": "^1.1.5",
     "yargs": "^9.0.1",
     "yeoman-environment": "^2.0.0",
-    "yeoman-generator": "git://github.com/ev1stensberg/generator.git#Feature-getArgument"
+    "yeoman-generator": "https://github.com/ev1stensberg/generator.git#Feature-getArgument"
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",

